<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QModding</class>
 <widget class="QSubDialogBase" name="QModding">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>生成模型</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEdit_4"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="lineEdit_6"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>面角度（double）</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>kd树k值（int）</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>搜索半径（double）</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>加权因子（double）</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;http://blog.csdn.net/u012337034/article/details/37924523&quot;&gt;&lt;span style=&quot; font-size:18pt; text-decoration: underline; color:#aa007f;&quot;&gt;说明&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>应用</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEdit_5"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>临近点阈值（int）</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="lineEdit_7"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>三角形最小角度（double）</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>三角形最大角度（double）</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QSubDialogBase</class>
   <extends>QWidget</extends>
   <header>qsubdialogbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>QModding</receiver>
   <slot>OnApply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnApply()</slot>
 </slots>
</ui>
